components/kyma-environment-broker/internal/ias/client.go

```diff
@@ -41,6 +41,7 @@ type (
 		Path    string
 		Body    io.Reader
 		Headers map[string]string
+		Delete  bool
```

Comment:
maybe we can have consistent naming with edp and also use `allowNotFound`

Reply:
I thought about this parameter more like a request method (Delete: true), I need to recognize if it is delete action, for IAS API it is just PUT method on `/service/sps/delete` endpoint.